doc/files/npmrc.md

```diff
@@ -41,6 +41,15 @@ sensitive credentials, they must be readable and writable _only_ by your user
 account (i.e. must have a mode of `0600`), otherwise they _will be ignored by
 npm!_
 
+#### Comments
+
+Because lines in an npm config file follow the `key = value` pattern, lines may be effectivly commmented-out by placing any character at the beginning of the line (creating a `key` which is not recoginized, or used). Common comment characters are `;`, `#`, or `//`.
```

Comment:
`;` and `#` represent actual comments which won't end up in the key-space for loaded config, by contrast `//` does not and in fact can be a part of valid keys. For instance, your authentication key is stored as:

```
//registry.npmjs.org/:_authToken=AUTH-TOKEN-HERE
```